src/api/ApiResponder.php

```diff
@@ -144,7 +109,7 @@ protected function get_interface_response_createGame($interface, $args) {
         if (isset($retval)) {
             foreach ($playerIdArray as $playerId) {
                 if (isset($playerId)) {
-                    $interface->update_last_action_time($playerId, $retval['gameId']);
+                    $interface->player()->update_last_action_time($playerId, $retval['gameId']);
```

Comment:
It's correct that the interface is already a BMInterfacePlayer object. However, it's not a costly cast, and it keeps the code consistent.

The point is that the programmer shouldn't need to know what sort of interface is being used, rather, each time a certain interface function is being called, the only important thing to know is what sort of interface provides that function.